medusa/tv/series.py

```diff
@@ -64,20 +49,20 @@
     qualityPresets,
     statusStrings,
 )
+from medusa.exceptions import (
+    RemovalError,
+    EpisodeNotFoundException,
+    IntegrityError,
+    IntegrityError,
```

Comment:
With all the refactors I was going to close this PR and recreate it to avoid all the conflict resolution.  I was just leaving it open as a reminder to get back to it after all the big changes.